### ISSUE TITLE
contrast for h3 navigation element

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#713]
```
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1:1. Recommendation: change text colour to #fff.
```

code:

```
<h3 class="wdn_list_descriptor wdn-text-hidden">Navigation</h3>
```

This is related to #704 but should probably a specific rule
